apps/paymaster-proxy/src/errors/PaymasterError.ts

```diff
@@ -0,0 +1,22 @@
+// These are the errors that can be proxied and returned to the user
```

Comment:
These objects will capture the errors that an underlying paymaster service can return. Right now it's pretty simple, but in the future we can add more detailed cases.